io-sim/src/Control/Monad/IOSim.hs

```diff
@@ -95,6 +95,7 @@ data SimA s a where
   Catch        :: Exception e =>
                   SimA s a -> (e -> SimA s a) -> (a -> SimA s b) -> SimA s b
 
+  GetTid       :: (ThreadId -> SimA s b) -> SimA s b
```

Comment:
This is here already as `GetThreadId` below.

Reply:
deleted